Rakefile

```diff
@@ -122,14 +122,6 @@ task :unit do
   run_command('rspec')
 end
 
-desc 'Multi-node Integration with Vagrant'
-task :multi_node do
-  run_command('rm -rf cookbooks nodes Berksfile.lock')
-  run_command('berks vendor cookbooks')
-  run_command('mkdir nodes')
-  run_command('vagrant up')
-end
-
```

Comment:
Why was this removed?

Reply:
IDK, I think in response to cookstyle, but it was probably an oversite by me, ill fix it

Reply:
@ramereth this is a github glitch. This change is already in master

Reply:
Nope, it shows up in 7435397b4e81082a1d03cb62767807e14c39fc2b

Reply:
Oh my goodness 🤦 I was looking at osl-ganeti so it didnt show up in my git diff